ministryofjustice/laa-apply-for-criminal-legal-aid
# Add correspondence address type + amend view + specs

## Description of change

Adds ability for a provider to choose a correspondence type for the address their applicant can be contacted on.

## Link to relevant ticket
[CRIMAP-58](https://dsdmoj.atlassian.net/browse/CRIMAP-58)

## Notes for reviewer
- this PR does not change the client decision tree. Changes to this will be added in a future PR.

## Screenshots of changes (if applicable)

### Before changes:
![Screenshot 2022-08-09 at 16 30 13](https://user-images.githubusercontent.com/13377553/183692841-96f0984e-00e1-44ba-923a-5b7222adbec4.png)

### After changes:
![Screenshot 2022-08-09 at 16 31 23](https://user-images.githubusercontent.com/13377553/183693328-4af44a02-771b-4104-a924-8ee5a14e3b8f.png)

## How to manually test the feature
- Go through journey past the address entry pages to the contact details page
- Test validations by not selecting a correspondence address type > error should appear
- Add a phone number and select an address and the page should progress to the holding page, pending decision tree updates in future PR
